2017-06-17 22:49:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
select name from user [42102-192]
2017-06-17 22:49:55 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
select name from user [42102-192]
2017-06-17 22:49:56 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST" not found; SQL statement:
select dsc from test [42102-192]
2017-06-17 22:49:56 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST" not found; SQL statement:
select dsc from test [42102-192]
2017-06-17 22:49:56 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST" not found; SQL statement:
select dsc from test [42102-192]
2017-06-17 22:49:57 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST" not found; SQL statement:
select dsc from test [42102-192]
2017-06-17 22:49:57 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST" not found; SQL statement:
select dsc from test [42102-192]
2017-06-17 22:49:57 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST" not found; SQL statement:
select dsc from test [42102-192]
2017-06-17 22:49:57 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
select name from user [42102-192]
2017-06-17 22:49:57 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
select name from user [42102-192]
2017-06-17 22:49:57 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
select name from user [42102-192]
2017-06-17 22:49:58 jdbc[14]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
select name from user [42102-192]
2017-06-17 22:49:58 jdbc[15]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
select name from user [42102-192]
2017-06-17 22:49:58 jdbc[16]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
select name from user [42102-192]
2017-06-17 22:49:58 jdbc[17]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
select name from user [42102-192]
2017-06-17 23:59:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST" not found; SQL statement:
select dsc from test [42102-192]
2017-06-17 23:59:05 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
select name from user [42102-192]
2017-06-18 00:02:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
select name from user [42102-192]
2017-06-18 00:02:11 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST" not found; SQL statement:
select dsc from test [42102-192]
2017-06-18 00:03:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST" not found; SQL statement:
select dsc from test [42102-192]
2017-06-18 00:03:02 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
select name from user [42102-192]
2017-06-18 00:05:14 jdbc[5]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-192]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:187)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:129)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebServlet.doGet(WebServlet.java:125)
	at org.h2.server.web.WebServlet.doPost(WebServlet.java:162)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:292)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:99)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.springframework.web.filter.HttpPutFormContentFilter.doFilterInternal(HttpPutFormContentFilter.java:87)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.springframework.web.filter.HiddenHttpMethodFilter.doFilterInternal(HiddenHttpMethodFilter.java:77)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:121)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:212)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:141)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:528)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1099)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:670)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1520)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1476)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 51 more
2017-06-18 00:05:14 jdbc[5]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-192]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:187)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:129)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebServlet.doGet(WebServlet.java:125)
	at org.h2.server.web.WebServlet.doPost(WebServlet.java:162)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:292)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:99)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.springframework.web.filter.HttpPutFormContentFilter.doFilterInternal(HttpPutFormContentFilter.java:87)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.springframework.web.filter.HiddenHttpMethodFilter.doFilterInternal(HiddenHttpMethodFilter.java:77)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:121)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:212)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:141)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:528)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1099)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:670)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1520)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1476)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 51 more
2017-06-18 00:40:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
select name from user [42102-192]
2017-06-18 00:50:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
select name from user [42102-192]
2017-06-18 01:26:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" already exists; SQL statement:
CREATE TABLE USER(ID INT PRIMARY KEY, 
                                           NAME VARCHAR(32) NOT NULL,
                                           ROLE INT NOT NULL,
                                           SALT VARCHAR(32),
                                           PASSWORD VARCHAR(64)) [42101-192]
